runtime/dotnet/core/Settings/BotSkillSettings.cs

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.BotFramework.Composer.Core.Settings
+{
+    public class BotSkillSettings
+    {
+        // Is skill
```

Comment:
Minor, optional: Ideally we strive for either xml docs (`///` format) or no docs.